cypress/integration/route_spec.js

```diff
@@ -0,0 +1,98 @@
+describe('Sea route tests', function () {
+
+    before (function () {
+	cy.login();
+    });
+
+    beforeEach(function () {
+        // Session cookies will not be cleared before the NEXT test starts
+        cy.preserveSessionOnce();
+    });
+
+
+    it ("Create new route", function () {
+	cy.visit ("/ote/#/routes/");
+
+	cy.wait(500);
+
+	cy.get('tr').should('have.length', 1);
+
+	cy.contains(/Lisää uusi merireitti/).click();
+    });
+
+    it ("Add basic information", function () {
+	cy.get('[id*="Reitinnimi"]').type("Test route").should('have.value', "Test route");
+	cy.get('[id*="Palveluntuottaja"]').click();
+	cy.contains(/Testinen/).click();
+	cy.get('[id*="departure-point-name"]').type("Oulu").should('have.value', "Oulu");
+	cy.get('[id*="destination-point-name"]').type("Vaasa").should('have.value', "Vaasa");
+	cy.get('[id*="undefined--Voimassaalkaen"]').click();
+	cy.get(':nth-child(3) > :nth-child(4) > span').click();
+	cy.get('[id*="undefined--Voimassaasti"]').click();
+	cy.get(':nth-child(5) > :nth-child(4) > span').click();
+    });
+
+    it ("Add stops", function () {
+	cy.get('img[title="OULUN SATAMA: Vihreäsaari"]').click();
+	cy.get('img[title="RAAHE: Rautaruukki"]').click();
+	cy.get('img[title="KALAJOKI: Itäkenttä"]').click();
+	cy.get('img[title="VAASA: Lassenlaituri"]').click();
+    });
+
+    it ("Add calendar", function () {
```

Comment:
Does this select any days from year calendar directly?

Reply:
No it doesn't. Only starting and ending days.